### DISCUSSION TITLE
Extra key-presses counted on key overlay

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#26602] [Ideas]
I'm suggesting that key-presses during beatmap intro time and when clicking the orange resume gameplay cursor be changed to not count in the key overlay to match stable.

**Comment by bdach** [comment]
> key-presses during beatmap intro time

These should already not count. Please attach video of this.

> and when clicking the orange resume gameplay

Covered by https://github.com/ppy/osu/issues/9658.

**Reply by ghost** [reply]
Sure, here is a video comparison. In the stable recording, hits are not counted in the key overlay until the first object appears. Clicking the orange resume gameplay button is not registered in stable as a hit on the key overlay, whilst it is in lazer.

Lazer:
https://github.com/ppy/osu/assets/129146357/fbc87f57-7426-4263-8ff0-af6b84644be0

Stable:
https://github.com/ppy/osu/assets/129146357/872ef417-a87c-42d0-9ca7-fed3ddfcbeaa
